### PR TITLE
Add switch documentation to bmw_connected_drive

### PR DESCRIPTION
**Pull request by rikroe**
## Proposed change

Add documentation for https://github.com/home-assistant/core/pull/92962.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/92962
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by gerard33 — CHANGES_REQUESTED** [review]
The `button.<your_vehicle>_deactivate_air_conditioning` is still mentioned [here](https://github.com/home-assistant/home-assistant.io/pull/27365/files#diff-cbd0dcc012c871642b6ab81d62d0f7f3aeb1edaf49d9937e06752f6441784085L107-L109). That can be removed.

File: source/_integrations/bmw_connected_drive.markdown [hunk 1]
```diff
@@ -144,9 +147,16 @@ If you have a (PH)EV, you can control the charging process through Home Assistan
 Using these selects will impact the state of your vehicle. Use them with care!
 
 - **Charging Mode**: Vehicle can be set to `IMMEDIATE_CHARGING` (charge as soon as plugged in) or `DELAYED_CHARGING` (charge only if within charging window). It can be used to start/stop charging if the charging window is set accordingly.
-- **Target SoC**: The vehicle will charge until this battery level is reached. Not available on all EVs.
```
Comment (gerard33) [inline source/_integrations/bmw_connected_drive.markdown:147]: You can mention that Target SoC is now part of the Number platform if available for the car

Comment (rikroe) [reply source/_integrations/bmw_connected_drive.markdown:147]: That PR was directly merged to `current` (https://github.com/home-assistant/home-assistant.io/pull/27147), so we have a small diversion at the moment. branches should be updated with the next major release.

**Review by frenck — APPROVED** [review]
Thanks, @rikroe 👍 

../Frenck

**Pull request merged**